js_modules/dagster-ui/packages/ui-components/src/components/BaseTag.tsx

```diff
@@ -47,8 +52,8 @@ export const BaseTag = (props: Props) => {
           data-tooltip={typeof label === 'string' ? label : tooltipText}
           data-tooltip-style={JSON.stringify({
             ...BaseTagTooltipStyle,
-            backgroundColor: fillColor,
-            color: textColor,
+            backgroundColor: colorTooltipBackground(),
+            color: colorTooltipText(),
```

Comment:
We should remove the props on line 39/40 if this is no longer customizable

Reply:
Those values are still used for the fillColor and textColor of the tag itself.